### ISSUE TITLE
Item trends

### ISSUE THREAD

**Issue by Kaptard** [nexus-devs/NexusHub#66]
The frontpage, as well as search results and similar pages would heavily benefit from the display of item trends. **Use this thread to post any useful ideas for trends**. 

Starting off:
- Highest demand increase relative to supply change (e.g. +120% demand with supply only at +10%, indicating a good niche to trade in)
- Highest price increase in last day (minimum of ~100 priced orders for accuracy)
- Most popular (highest demand + supply + page views)
- Highest price with highest demand (super useful for sellers)

**Comment by TobiTenno** [comment]
* Median prices over a week
* Median price changes since vaulting (if vaulted)

**Comment by DelSquared** [comment]
Maybe a future price prediction (be sure to include a **HUGE** disclaimer that it is just an estimate tho lol). I am currently working on some techniques based off the data from warframe.market that I might put up (sadly in Python but I guess the process is what's important). 

So far I have set up a Monte-Carlo simulation and was considering implementing a recurring neural net but the data available on WM alone might not be enough. However the data that you have (like supply-demant ratios, in-game availability such as un/vaultings, trade-chat population, active users, prices, etc) and other data that can be calculated from this could be enough to deploy a RNN that's remotely useful (provided the computing power is adequate)

**Comment by Kaptard** [comment]
@DelSquared Experimental price predictions will be part of the core price graphs sometime in the future, although applying that data to trends is also an interesting idea. 👍

**Comment by DelSquared** [comment]
Coincidentally Google just launched [Tensorflow.js](https://js.tensorflow.org/)
